docs/guides/setup/installation/local.md

```diff
@@ -6,41 +6,97 @@ import Congratulations from '/docs/guides/setup/components/install_congrats.md'
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-This guide will help you Install Great Expectations locally for use with Python
+This guide will help you Install Great Expectations locally for use with Python.
+
+:::note
+- Great Expectations is developed and tested on macOS and Linux Ubuntu. Installation for Windows users may vary from the steps listed below. If you have questions, feel free to reach out to the community on our [Slack channel](https://greatexpectationstalk.slack.com/join/shared_invite/zt-sugx45gn-SFe_ucDBbfi0FZC0mRNm_A#/shared-invite/email).
+- If you have the Mac M1, you may need to follow the instructions in this blog post: [Installing Great Expectations on a Mac M1](https://greatexpectations.io/blog/m-one-mac-instructions/).
+:::
+
 
 ## Steps
 
 :::note Prerequisites
 - Great Expectations requires Python 3. For details on how to download and install Python on your platform, see [python.org](https://www.python.org/downloads/).
 :::
 
-### 1. Install Python
+### 1. Check Python version
 
-First, check that you have Python 3 with pip installed. You can confirm your version of python with `python --version`.
+First, check that you have Python 3 installed. You can check your version of Python by running:
 
-```bash
-# if python is version 3
+```console
 python --version
-python -m ensurepip --upgrade
+```
+
+If this command returns something other than a Python 3 version number (like Python 3.X.X), you may need to try this:
 
-# or if multiple versions of Python installed
+```console
 python3 --version
-python3 -m ensurepip --upgrade
 ```
 
 ### 2. Install Great Expectations
 
-Use pip to install Great Expectations.
+You can use either pip or Anaconda to install Great Expectations.
 
-```bash
-# if python is version 3
+<Tabs
+  groupId="pip-or-conda"
+  defaultValue='pip'
+  values={[
+  {label: 'pip', value:'pip'},
+  {label: 'conda', value:'conda'},
+  ]}>
+<TabItem value="pip">
+
+Once you have confirmed that Python 3 is installed locally, you will run two commands that will:
+
+1. Ensure that pip is installed and upgraded to the latest version
+2. Install Great Expectations using pip
+
+Depending on whether you found that you needed to run `python` or `python3` in the previous step, you will run either:
+
```

Comment:
Do you think we should advise to install ge into a virtualenv for pip, to be consistent with the conda instructions?

Reply:
I have added instructions for installation into a virtual environment using `venv`.